terraform/groups/configurable-api-caller/locals.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+
+  stack_name                   = "utility"
+  lambda_env_vars              = merge(local.service_secrets, var.open_lambda_environment_variables)
+  lambda_vpc_access_subnet_ids = data.aws_subnets.application.ids
+  json_folder                  = "profiles/${var.aws_account}/common-${var.aws_region}"
```

Comment:
These files should not be in the `profiles/` directory structure. I would suggest creating a new directory structure under `groups/configurable-api-caller`

E.g. something like `groups/configurable-api-caller/input_json`, or similar.

Please also be sure to review the input JSON for sensitive data